test/e2e/copy_from_image_test.go

```diff
@@ -278,6 +245,40 @@ func TestCopyToTarWithoutNonDistributableLayersFlagButContainsANonDistributableL
 			t.Fatalf("Expected non-distributable layer to NOT be copied into registry, however it was")
 		}
 	})
+
+	t.Run("With --lock-output flag should generate a valid ImageLock file", func(t *testing.T) {
```

Comment:
is this related to perf work?

Reply:
No, that was just a test being re-organized. running it in a t.Run allowed it to belong to a parent context and for the test name to be a sentence.